lib/capistrano/tasks/assets.rake

```diff
@@ -62,15 +62,29 @@ namespace :deploy do
       end
     end
 
+    def manifest_path(filename)
+      release_path.join('public', fetch(:assets_prefix), filename)
+    end
+
+    def with_backup_and_manifest_paths &block
```

Comment:
There is `with` method to set env in target machine. This method name can be confusing without merit.